src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -128,37 +129,48 @@ void render(FlutterLogEntry entry) {
     }
 
     private class MessageCellRenderer extends EntryCellRenderer {
-      @Nullable
-      private final FlutterConsoleFilter consoleFilter;
+      @NotNull
+      private final FlutterApp app;
+      @NotNull
+      private final List<Filter> linkFilters;
```

Comment:
(See below but I wonder if we might not migrate to a speedier native array here.)

Reply:
Yeap, I changed to: `private final Filter[] filters;`